aggregator_core/src/datastore.rs

```diff
@@ -1239,6 +1253,8 @@ impl<C: Clock> Transaction<'_, C> {
                 /* task_id */ task_id.as_ref(),
                 /* report_id */ &report_id.get_encoded(),
                 /* now */ &self.clock.now().as_naive_date_time()?,
+                /* updated_at */ &self.clock.now().as_naive_date_time()?,
+                /* updated_by */ &self.name,
             ],
         )
         .await?;
```

Comment:
This block of code (1204-1226) could be:
```rust
        let stmt = self
            .prepare_cached(
                "UPDATE client_reports
                SET aggregation_started = false
                SET aggregation_started = false, updated_at = $3, updated_by = $4
                FROM tasks
                WHERE client_reports.task_id = tasks.id
                  AND tasks.task_id = $1
                  AND client_reports.report_id IN (SELECT * FROM UNNEST($2::BYTEA[]))
                  AND client_reports.client_timestamp >= COALESCE($3::TIMESTAMP - tasks.report_expiry_age * '1 second'::INTERVAL, '-infinity'::TIMESTAMP)",
            )
            .await?;
        let row_count = self
            .execute(
                &stmt,
                &[
                    /* task_id */ &task_id.as_ref(),
                    /* report_ids */ &report_ids,
                    /* now, updated_at */ &self.clock.now().as_naive_date_time()?,
                    /* updated_by */ &self.name,
                ],
            )
            .await?;
```
Alternatively, if wanting to avoid having to keep track of argument indexes, we can have `let now = &self.clock.now().as_naive_date_time()` and supply `&now` to the args slice where necessary.

I don't feel strongly about this though, it's ultimately a minor refactor.